source/manual/alerts/email-alerts-travel-medical.html.md

```diff
@@ -31,23 +31,31 @@ the message: `govuk_email_check@digital.cabinet-office.gov.uk`. Its credentials
 can be found in the [2nd line password store][] under
 `google-accounts/govuk_email_check@digital.cabinet-office.gov.uk`.
 
-If the email has been received by the mailbox but the subject of the email
-doesn't match the title of the content item, this means that the title of the
-content was updated after it was first published and after the emails went out.
-To stop the check from constantly failing, add the updated content item title
-to the [acknowledged email list][] and then re-run the Jenkins job.
+Previous false positives of this alert have been:
```

Comment:
This is a bit strange to read after the troubleshooting steps, but valuable information for someone debugging the alert. How about putting it in its own section - like "DEPRECATED: be aware of false positives" - with a link to the ADR?

Reply:
I think I'm interpreting your usage of deprecated to mean that this alert type is deprecated, but your suggestion would make it sound like the things to look out for are deprecated. I think it'd make more sense to say something like.


> These alerts are vulnerable to false positives, and indeed there is a [proposal](https://github.com/alphagov/email-alert-api/blob/master/docs/adr/adr-008-monitoring-and-alerting.md) to replace this alert, previous ones have included: 

I should also add the time Gmail had an outage to the list.

Reply:
That sounds good, but is still at odds with the previous two paragraphs: the first and third bullet points are another perspective on the previous paragraph, while the second is unrelated. Since the whole section seems to be about false positives (there's no mention of actual technical troubleshooting), perhaps we can merge the content together more, and perhaps retitle the section as "Check for false positives" or similar?

Reply:
Thanks. I've moved bits around here somewhat - it's now 3 sections, with the medical check covered as "not sent problem"